torchrl/objectives/ppo.py

```diff
@@ -71,6 +73,19 @@ class PPOLoss(LossModule):
             policy and critic will only be trained on the policy loss.
             Defaults to ``False``, ie. gradients are propagated to shared
             parameters for both policy and critic losses.
+        kl_init_coef (scalar, optional): A coefficient for the KL-divergence
+            between the current actor configuration and the initial one.
+            If ``kl_init_coef > 0``, a ``init_actor_params`` attribute
+            will be available and contain the initial configuration of the
+            parameters for the actor. These will be used to return a ``"loss_kl_init"``
```

Comment:
> ``init_actor_params`` attribute
            will be available and contain the initial configuration of the
            parameters for the actor
            
not super clear. what do you mean?

Reply:
The property won't be available otherwise

Reply:
Right! I usually don't define docstrings for properties bc I don't expect users to know that smth is a property and not a regular attribute (hence they will never call `help(obj.smth)`)
Can we write it in both places?

Reply:
I think here it is not very useful. If you want to keep it I would just write something short as `If kl_init_coef > 0, the init_actor_params will raise`

another option is to make the attribute private.